.vscode/settings.json

```
@@ -0,0 +1,3 @@
+{
+    "rust-analyzer.cargo.features": ["unstable"]
```

I develop locally with:
```suggestion
    "rust-analyzer.cargo.features": "all"
```